### PR TITLE
Gather fits

### PR DESCRIPTION
**Pull request by MaxNoe**
* Add script to perform datacheck and join corresponding outputfiles into one (h5py) hdf5

The hdf5 contains a group `events` with the features from the fits files, a group `runs` with the columns
night, run_id, ontime, zenith for the selected runs und the query is stored in the metadata:
```
f = h5py.File('my_erna_output.hdf5')
print(f['runs'].attrs['datacheck'])
```

* Refactor erna.load to use the same functions as gather fits (do the datacheck queries on the db, not download the complete db and then do it)

* Refactor datacheck conditions (Removed all but the most basic, as most of the stuff was commented out or contained errors). We might need to add stuff back here.

### CODE REVIEW

File: erna/scripts/gather_fits.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+@click.command()
+@click.argument('xml-name')
+@click.argument('ft-version')
+@click.argument('outputfile')
+@click.option('--config', '-c')
+@click.option('--start', '-s', help='First night to get data from')
+@click.option('--end', '-e', help='Last night to get data from')
+@click.option('--source', default='Crab')
+@click.option('--datacheck', help='The name of a condition set for the datacheck')
+def main(xml_name, ft_version, outputfile, config, start, end, source, datacheck):
```
Comment (MaxNoe) [inline erna/scripts/gather_fits.py]: It has now

File: erna/utils.py [hunk 2]
```diff
@@ -51,3 +52,11 @@ def chown(path, username=None, groupname=None):
     uid = pwd.getpwnam(username).pw_uid if username else -1
     gid = grp.getgrnam(groupname).gr_gid if groupname else -1
     os.chown(path, uid, gid)
+
+
+def night_int_to_date(night):
```
Comment (MaxNoe) [inline erna/utils.py]: done

**Comment by MaxNoe** [comment]
I implemented the requested changes

**Comment by mackaiver** [comment]
:+1:

**Pull request merged**